### ISSUE TITLE
False report `SlevomatCodingStandard\Sniffs\Namespaces\UnusedUsesSniff`

### ISSUE THREAD

**Issue by TomasVotruba** [slevomat/coding-standard#106]
## Code

```php

use Symplify\EasyCodingStandard\Report\Error\Error;

class ...
{

    /**
     * @return Error[]
     */
    private function getRelevantErrorMessages(bool $isFixer): array
    {
        // ...
    }
}
```

## Report

![image](https://cloud.githubusercontent.com/assets/924196/23144099/a730938a-f7c5-11e6-854d-803150a83e5d.png)

**Comment by kukulich** [comment]
@TomasVotruba Did you enabled `$searchAnnotations` property for this sniff in your ruleset.xml?

**Comment by ondrejmirtes** [comment]
Please make sure that you have set `searchAnnotations` property to true (https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesunuseduses-) and also that you are using the latest version available in the `2.0-dev` branch.

**Comment by TomasVotruba** [comment]
It works, thank you. Why it is disabled by default?

**Comment by kukulich** [comment]
Because we prefer to have FQN in annotations.

**Comment by TomasVotruba** [comment]
I see. Why so?
